docs/changelog/1_0_a3.rst

```diff
@@ -0,0 +1,137 @@
+
+* Add ``+=`` and ``-=`` operations for use in :eql:stmt:`UPDATE`
+  (:eql:gh:`#165`).
+* Distinguish ``REQUIRED`` and ``OPTIONAL`` query parameters
+  (:eql:gh:`#1352`, :eql:gh:`#1355`)
+* Allow tuple arrays in schema definitions (:eql:gh:`73125882`).
+* Get rid of ``schema::CompoundType`` in favor of ``is_compound_type``
+  property (:eql:gh:`c30c1c4f`).
+* Add :eql:type:`anytype` and :eql:type:`anytuple` as proper abstract
+  types to the schema (:eql:gh:`affb65de`).
+* ``schema::Constraint`` no longer has ``args``, use ``params`` instead
+  (:eql:gh:`14cec6f1`).
+* Make ``std::Object`` the implicit base type for all user-defined
+  object types only.
+* Make ``std::BaseObject`` the root type for all object types both
+  user-defined and system object types.
+* Accept ``1e100n`` format as a valid ``bigint`` constant rather than
+  ``decimal`` (:eql:gh:`#1372`)
+* Make self-referencing (recursive) aliases forbidden.
+* Fix dependency tracking affecting index creation (:eql:gh:`#1181`).
+* Fix derivation of link targets in certain cases of multiple
+  inheritance (:eql:gh:`52c6b2d4`).
+* Fix handling of ad-hoc tuples (:eql:gh:`#1255`).
+* Fix incorrect implicit limit injection in subqueries in computables
+  (:eql:gh:`#1271`).
+* Computables cardinality must now be declared explicitly as
+  ``required``, ``single`` or ``multi``. The expression is validated
+  to be within the upper and lower limits implied by the declaration
+  (:eql:gh:`#1201`, :eql:gh:`#349`).
+* Prohibit database names longer than 63 characters (:eql:gh:`#1158`).
+* Prohibit duplicate elements in shapes (:eql:gh:`#1368`).
+
+
+GraphQL
+=======
+
+* Reflect "description" annotations into GraphQL descriptions
+  (:eql:gh:`#1228`).
+* Change the auto-generated type name template to use "\_Type" suffix
+  (:eql:gh:`#1175`).
+* Fix incorrect reflection of enums (:eql:gh:`#1227`).
+* Fix deeply nested insert issue (:eql:gh:`#1243`).
+
+
+Standard Library
+================
+
+* Add ``sys::get_current_database()``
+* Rename ``std::to_str()`` array-joining function to
+  :eql:func:`array_join`.
+* Rename ``std::to_array()`` string-splitting function to
+  :eql:func:`str_split`.
+
+
+Server
+======
+
+* Convert EdgeQL lexer (:eql:gh:`#1178`)
+* Convert GraphQL parser to Rust and perform constant extraction
+  (:eql:gh:`#1299`).
+* Constant extraction from EdgeQL queries (:eql:gh:`#1356`)
+* Make it possible to use different EdgeDB servers connected to the
+  same Postgres cluster (:eql:gh:`#1197`).
+* Protocol: implement ``JSON_ELEMENTS`` IO format for responses
+  (:eql:gh:`#1169`).
+
+
+
+Misc
+====
+
+* Add cheatsheet to documentation.
+* Strictly type-annotate the core schema module.
+* Clean-up and improve schema and introspection code to make the
+  internal APIs more consistent (:eql:gh:`#1408`).
+* Update the `edgedb-js <https://github.com/edgedb/edgedb-js>`_ driver
+  to v0.7.3.
+* Update the `edgedb-python <https://github.com/edgedb/edgedb-python>`_
+  driver to v0.8.0a1.
```

Comment:
0.8.0, without a1.  We'll release a stable version today/tomorrow.

Reply:
OK